common/services/prismic/parsers.js

```diff
@@ -317,6 +317,16 @@ export function parseBody(fragment: PrismicFragment[]) {
           value: slice.primary.text
         };
 
+      case 'map':
+        return {
+          type: 'map',
+          value: {
+            title: asText(slice.primary.title),
```

Comment:
I know it __might__ be useful, but given we have so much unused stuff, is title necessary?

Reply:
It's used as the title attribute for the map marker